ROADMAP.md

```diff
@@ -147,19 +147,26 @@ Any dates listed below and the specific issues that will ship in a given milesto
   * [User experience for template stacks](https://github.com/crossplaneio/crossplane/blob/master/design/design-doc-template-stacks-experience.md).
 * [Bug fixes and other closed issues](https://github.com/crossplaneio/crossplane/milestone/6?closed=1)
 
-## v0.7.0
+## [v0.7.0 Deploy Workloads to any Kubernetes Cluster, including bare-metal!](https://github.com/crossplaneio/crossplane/releases/tag/v0.7.0)
 * KubernetesTarget kind for scheduling KubernetesApplications [#859](https://github.com/crossplaneio/crossplane/issues/859)
+* Improved the UI schema for resources supported by Crossplane stacks [#38](https://github.com/upbound/crossplane-graphql/issues/38)
+* GCP integration tests [crossplaneio/stack-gcp#87](https://github.com/crossplaneio/stack-gcp/issues/87)
+* Template Stacks (experimental): integrate template engine controllers with stack manager [#36](https://github.com/upbound/stacks-marketplace-squad/issues/36)
+
+## v0.8.0
```

Comment:
how about the design work that is ongoing for defining your own claim kinds #1106 and allowing a claim to be satisfied by multiple resources #1105

Reply:
done